### ISSUE TITLE
parser can only handle absolute paths to a raml definition file

### ISSUE THREAD

**Issue by mutsys** [raml-org/raml-java-parser-2#11]
passing anything other than an absolute path to a raml file results in errors in the bundled javascript

this path reliably fails:

```
src/main/raml/example.raml
```

with the following exception:

```
javax.script.ScriptException: TypeError: null has no such function "isTopLevel" in <eval> at line number 4942
```

this path reliably fails:

```
~/raml/raml-1/src/main/raml/example.raml
```

with the following exception:

```
javax.script.ScriptException: TypeError: null has no such function "title" in <eval> at line number 97
```

the following path reliably succeeds:

```
/Users/mrose/raml/raml-1/src/main/raml/example.raml
```

**Comment by sichvoge** [comment]
Hi @mutsys, this parser is a wrapper for the JS parser and we realised that there are couple of problems going this root. Therefore, we decided to implement a native Java parser which we will have a beta in a couple of weeks. stay tuned

**Comment by jschamburger** [comment]
Hi @sichvoge, are there any news on this? I'd love to get my hands on the new Java parser. :-)

**Comment by sichvoge** [comment]
Yes, we are currently finalising the alpha version, fixing couple of functional bugs, doing some refactoring, and soon publishing it. I will let you know where we are by the end of this week.

**Comment by jschamburger** [comment]
Great, thanks. Looking forward to it!
